rest/client/client.go

```diff
@@ -109,7 +109,6 @@ func (c *communicatorImpl) resetClient() {
 	}
 
 	conf := utility.NewDefaultHTTPRetryConf()
-	conf.Statuses = append(conf.Statuses, http.StatusBadRequest)
 	c.httpClient = utility.GetHTTPRetryableClient(conf)
```

Comment:
use `GetDefaultHTTPRetryableClient` and remove line 111
https://github.com/evergreen-ci/utility/blob/master/http.go#L184